docs/elements/http.html

```diff
@@ -0,0 +1,664 @@
+    </p>
+</article-paragraphs>
+
+<code-snippet
+        gherkin="Then the response status should be gone"
+        php="
+    /**
+     * @Then the response status should be gone
+     */
+    public function aResponseStatusShouldBeGone(): void"></code-snippet>
+
+<article-paragraphs>
+    <p>
+        The <code>aResponseStatusShouldBeServiceUnavailable</code> method verifies that the response status is HTTP 503
+        Service Unavailable. This is useful for ensuring that the service is temporarily unavailable.
+    </p>
+</article-paragraphs>
+
+<code-snippet
+        gherkin="Then the response status should be service unavailable"
+        php="
+    /**
+     * @Then the response status should be service unavailable
+     */
+    public function aResponseStatusShouldBeServiceUnavailable(): void"></code-snippet>
+
+<article-paragraphs>
+    <p>
+        The <code>aResponseStatusShouldBeFound</code> method verifies that the response status is HTTP 302 Found. This
+        is useful for ensuring that the resource was found and the request was redirected.
+    </p>
+</article-paragraphs>
+
+<code-snippet
+        gherkin="Then the response status should be found"
+        php="
+    /**
+     * @Then the response status should be found
+     */
+    public function aResponseStatusShouldBeFound(): void"></code-snippet>
+
+<article-paragraphs>
+    <p>
+        The <code>aResponseShouldBeRedirectedTo</code> method verifies that the response is a redirect to the specified
+        URL. This is useful for ensuring that the request was redirected to the correct location.
+    </p>
+</article-paragraphs>
+
+<code-snippet
+        gherkin="Then the response should be redirected to '/dashboard'"
+        php="
+    /**
+     * @Then the response should be redirected to :url
+     */
+    public function aResponseShouldBeRedirectedTo(string $url): void"></code-snippet>
+
+<article-paragraphs>
+    <p>
+        The <code>aResponseShouldHaveHeaderContaining</code> method verifies that a response header contains a specified
+        value. This is useful for ensuring that headers contain expected values.
+    </p>
+</article-paragraphs>
+
+<code-snippet
+        gherkin="Then the response should have a header 'Content-type' containing 'application/json'"
+        php="
+    /**
+     * @Then the response should have a header :header containing :value
+     */
+    public function aResponseShouldHaveHeaderContaining(string $header, string $value): void"></code-snippet>
+
+<article-paragraphs>
+    <p>
+        The <code>aResponseShouldHaveNoContent</code> method verifies that the response content is empty. This is useful
+        for ensuring that no content is returned in the response.
+    </p>
+</article-paragraphs>
+
+<code-snippet
+        gherkin="Then the response should have no content"
+        php="
+    /**
+     * @Then the response should have no content
+     */
+    public function aResponseShouldHaveNoContent(): void"></code-snippet>
+    
```

Comment:
Here GitHub shows that there is no empty line at the end of the file.

![image](https://github.com/blumilksoftware/blt/assets/56546832/5198e055-3b91-46e3-a5bd-a9104b78f105)